tests/README.rst

```diff
@@ -80,6 +80,125 @@ To do that, run `podman pull quay.io/rhinstaller/anaconda-ci:master` or build
 it locally again.
 
 
+Explanation of the GitHub configuration
+---------------------------------------
+
+Pull request for master:
+________________________
+
+Everything is stored in `.github/workflows/validate.yml`_ the workflow.
```

Comment:
You need to actually define the target of link, otherwise it's just dangling and will be [rendered broken](https://github.com/jkonecny12/anaconda/blob/master-describe-GH-CI-solution/tests/README.rst#pull-request-for-master). See my [previous comment](https://github.com/rhinstaller/anaconda/pull/3041#discussion_r538204764) where I pointed to an example.

Also, "in the ... workflow", not "in ... the workflow".

Reply:
Great solution. Thanks!